src/boot/bootctl-random-seed.c

```diff
@@ -122,6 +158,8 @@ int install_random_seed(const char *esp) {
         } else {
                 ssize_t n;
 
+                warned = random_seed_verify_permissions(fd, S_IFREG) > 0;
```

Comment:
this could return a negative value too, shouldn't that be handled?

Reply:
not really, the whole purpose of the function is to print a warning to users, that's all. If that doesn't work we aren't too concerned and just continue. It's auxiliary to the main operation we want to execute here: the actual random seed refresh.

The only reason we care for the return value at all is that we only output one warning about the random seed file, not two (i.e. not one for the old and one for the new one).